src/Microsoft.ML/Models/SaveAsOnnx.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Model.Onnx;
+
+namespace Microsoft.ML.Models
+{
+    /// <summary>
+    /// Converts a model to ONNX format.
+    /// </summary>
+    public sealed class SaveAsOnnx
```

Comment:
>sealed [](start = 11, length = 6)

Something of a moot point because you ought to get rid of this anyway, but for a class made of static  methods note that there is a `static` keyword to apply to classes.